api/types/installers/installer.sh.tmpl

```diff
@@ -91,10 +91,10 @@ on_gcp() {
     JOIN_METHOD=iam
     LABELS="teleport.dev/instance-id=${INSTANCE_ID},teleport.dev/account-id=${ACCOUNT_ID}"
   elif on_gcp; then
-    INSTANCE_INFO=$(curl -m5 -sS -H "Metadata-Flavor: Google" "http://metadata.google.internal/computeMetadata/v1/instance/?recursive=true")
+    INSTANCE_INFO=$(curl -m5 -sS -H "Metadata-Flavor:Google" "http://metadata.google.internal/computeMetadata/v1/instance/?recursive=true")
 
-    NAME="$(echo "$INSTANCE_INFO" | jq -r .name)"
-    ZONE="$(echo "$INSTANCE_INFO" | jq -r .zone | awk -F "/" '{print $9}')"
+    NAME="$(curl -m5 -sS -H "Metadata-Flavor:Google" "http://metadata.google.internal/computeMetadata/v1/instance/name")"
+    ZONE="$(curl -m5 -sS -H "Metadata-Flavor:Google" "http://metadata.google.internal/computeMetadata/v1/instance/zone" | awk '{n=split($0,A,"/"); print A[n]}')"
```

Comment:
Here we should:
a) Add a comment that `zone` key returns a zone in the format `projects/PROJECT_NUM/zones/ZONE` so it needs parsing.
b) Instead of `awk` magic, can we just use `$(curl xxxx | basename)` since the format resembles a path?